SingularityUI/app/views/simpleSubview.coffee

```diff
@@ -20,18 +20,19 @@ class SimpleSubview extends View
 
         for eventName in ['sync', 'add', 'remove', 'change']
             @listenTo @data, eventName, @render
-            
+
         @listenTo @data, 'reset', =>
             @$el.empty()
 
     render: ->
         return if not @data.synced and @data.isEmpty?()
-        
+
         @$el.html @template
-            config:   config
-            data:     @data.toJSON()
-            synced:   @data.synced
-            expanded: @expanded
+            config:    config
+            data:      @data.toJSON()
+            synced:    @data.synced
+            expanded:  @expanded
+            modelData: @model?.toJSON()
```

Comment:
it's probably fine, but are we cool with this extra `toJSON()` call for every use of `simpleSubview`?